packages/compass/src/app/index.jsx

```diff
@@ -71,9 +71,7 @@ const Application = View.extend({
     return [
       '<div id="application">',
       '  <div data-hook="auto-update"></div>',
```

Comment:
I guess Maurizio will clean this further up, but if not, I will do it myself in the next pr.